extensions/pom.xml

```diff
@@ -71,7 +71,7 @@
                 <artifactId>quarkus-maven-plugin</artifactId>
                 <inherited>false</inherited>
                 <!-- Setting for stubbing new extensions via
-                       mvn quarkus:create-extension groovy:execute@scaffold-integration-test -N -Dquarkus.artifactIdBase=my-new-extension
+                       mvn quarkus:create-extension -N -Dquarkus.artifactIdBase=my-new-extension
```

Comment:
Add to the configuration of quarkus-maven-plugin below : `<itestParentPath>../integration-tests/pom.xml</itestParentPath>` - that will activate the itest module generation.

Reply:
done